.travis.yml

```diff
@@ -2,6 +2,10 @@
 # ref: https://blog.travis-ci.com/2015-10-14-opening-up-ubuntu-trusty-beta/
 sudo: required
 dist: trusty
+group: deprecated-2017Q4
```

Comment:
Can we add a comment above this explaining what it does?

Reply:
Yes!